main/src/ca/uwaterloo/flix/language/errors/NameError.scala

```diff
@@ -102,21 +97,25 @@ object NameError {
     * @param name the name of the type variable.
     * @param loc  the location of the suspicious type variable.
     */
-  case class SuspiciousTypeVarName(name: String, loc: SourceLocation) extends TypeNameError {
+  case class SuspiciousTypeVarName(name: String, loc: SourceLocation) extends NameError {
     def summary: String = s"Suspicious type variable '$name'. Did you mean: '${name.capitalize}'?"
 
     def message(formatter: Formatter): String = {
       import formatter._
       s"""${line(kind, source.name)}
+         |
          |>> Suspicious type variable '${red(name)}'. Did you mean: '${cyan(name.capitalize)}'?
          |
-         |${code(loc, "Suspicious type variable.")}
+         |${code(loc, "suspicious type variable.")}
          |""".stripMargin
     }
 
     def explain(formatter: Formatter): Option[String] = Some({
-      """Flix uses lowercase variables. The provided type variable looks suspiciously
-        |like the name of a built-in type. Perhaps you meant to use the built-in type?
+      """Flix uses lowercase variable name.
+        |
+        |The type variable looks suspiciously like the name of a built-in type.
+        |
+        |Perhaps you meant to use the built-in type?
```

Comment:
Would be nice if we named the type we think they want, but perhaps that's future work.

Reply:
Yeah